addons/sale/tests/test_sale_order.py

```diff
@@ -629,6 +629,55 @@ def test_cannot_assign_tax_of_mismatch_company(self):
         with self.assertRaises(UserError):
             sol.tax_id = tax_b
 
+    def test_assign_tax_multi_company(self):
+        root_company = self.env['res.company'].create({'name': 'B0 company'})
+        root_company.write({'child_ids': [
+            Command.create({'name': 'B1 company'}),
+            Command.create({'name': 'B2 company'}),
+        ]})
+
+        country = self.env['res.country'].search([])[0]
```

Comment:
```suggestion
        country = self.env['res.country'].search([], limit=1)
```
Would be nice to change it on line 598 too (The test above)